src/docs/asciidoc/Application-Configuration.asciidoc

```diff
@@ -397,6 +397,7 @@ security:
   basic:
     enabled: true                                                     # <2>
     realm: SpringXD                                                   # <3>
+    role: ADMIN, VIEW, CREATE                                         # <4>
```

Comment:
Yeah, sorry. The servers.yml entry is the correct one. I will fix the documentation.